actors/miner/tests/vesting_unvested_funds.rs

```diff
@@ -17,7 +15,8 @@ fn unlock_unvested_funds_leaving_bucket_with_non_zero_tokens() {
 
     h.add_locked_funds(vest_start, &vest_sum, &vspec).unwrap();
 
-    let amount_unlocked = h.unlock_unvested_funds(vest_start, &TokenAmount::from_atto(39)).unwrap();
+    let amount_unlocked =
+        h.unlock_vested_and_unvested_funds(vest_start, &TokenAmount::from_atto(39)).unwrap().0;
     assert_eq!(TokenAmount::from_atto(39), amount_unlocked);
```

Comment:
```suggestion
    let (unvested_unlocked, vested_unlocked) =
        h.unlock_vested_and_unvested_funds(vest_start, &TokenAmount::from_atto(39)).unwrap();
    assert_eq!(TokenAmount::from_atto(39), unvested_unlocked);
    assert_eq!(TokenAmount::zero(), vested_unlocked);
```

can we be complete in this test?